main.tex

```diff
@@ -379,12 +386,12 @@ \section{Results}
 Figure~\ref{fig:results} shows the results of our experiments. In this section, we discuss the results and how they relate to the three research questions outlined in the previous section.
 
 \subsection{Multiprocessing}
-With batch size 1, the testing speed scales almost linearly with the number of \acro{CPU}s, as we would expect. Whereas with with 1~\acro{CPU}, the median testing time is 27~hours and 2~minutes, it is only 1~hour and 6~minutes with 32~\acro{CPU}s (about 24-fold speed-up).\footnote{%
-The reason why we did not achieve the theoretical 32-fold speedup is likely due to tasks from other users on our server.%
+With batch size 1, the testing speed scales almost linearly with the number of \acro{CPU}s, as we would expect: with 1~\acro{CPU}, the median testing time is 27~hours and 2~minutes, it is only 1~hour and 6~minutes with 32~\acro{CPU}s (about 24-fold speed-up).\footnote{%
+The reason that we did not achieve the theoretical 32-fold speedup is likely tasks from other users on our server.%
 }
 
 \subsection{Batching of test files}
-With 1~\acro{CPU}, the testing speed also scales almost linearly with the batch size up to a point. Whereas with batch size 1, the median testing time is 27~hours and 2~minutes, it is only 4~hours and 30~minutes with batch size 8 (about 6-fold speed-up), and only 1~hour and 20~minutes with batch size 512 (about 21-fold speed-up). This indicates that initialization dominates the testing time.
+With 1~\acro{CPU}, the testing speed also scales almost linearly with the batch size, up to a point. Whereas with batch size 1, the median testing time is 27~hours and 2~minutes, it is 4~hours and 30~minutes with batch size 8 (about 6-fold speed-up), and 1~hour and 20~minutes with batch size 512 (about 21-fold speed-up). This indicates that initialization dominates the testing time.
```

Comment:
The removal of "only" is inconsistent with the previous section and also next section.

```suggestion
With 1~\acro{CPU}, the testing speed also scales almost linearly with the batch size, up to a point. Whereas with batch size 1, the median testing time is 27~hours and 2~minutes, it is only 4~hours and 30~minutes with batch size 8 (about 6-fold speed-up), and 1~hour and 20~minutes with batch size 512 (about 21-fold speed-up). This indicates that initialization dominates the testing time.
```